superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu

```diff
@@ -32,9 +32,15 @@ struct BenchArgs {
     // GPU IDs for worker device.
     int worker_gpu_id = 0;
```

Comment:
Why not worker_gpu_id[]? It looks like it cannot extend..

Reply:
I think an array here will make these code much easier. Let me change it.

Reply:
Fixed, now each benchmark will have several sub-benchmarks running in parallel. For example, a bidirectional benchmark consists of two unidirectional sub-benchmarks in parallel.